core/src/main/scala/org/bitcoins/core/protocol/dlc/models/ContractInfo.scala

```diff
@@ -220,11 +220,7 @@ case class SingleContractInfo(
   }
 
   /** @inheritdoc */
-  override val max: Satoshis = contractDescriptor match {
-    case descriptor: EnumContractDescriptor =>
-      descriptor.values.maxBy(_.toLong)
-    case _: NumericContractDescriptor => totalCollateral
-  }
+  override val max: Satoshis = totalCollateral
```

Comment:
doesn't this go against the whole PR's description of changing it not to be `max_payout = offer_collateral + accept_collateral`